src/Sylius/Bundle/ContentBundle/Resources/translations/messages.th.yml

```diff
@@ -88,9 +88,9 @@ sylius:
             title: หัวข้อ
         static_content:
             body: Body
-            id: ไอดี
+            id: ID
             internal_name: ชื่อภายใน
-            menu_nodes: เมนูต่างๆ
+            menu_nodes: Menu nodes
```

Comment:
I believe it's not SyliusBot fault 😄  just wondering if it's ok to accept such changes... /cc @pjedrzejewski